server/src/test/java/io/crate/integrationtests/TableSettingsTest.java

```diff
@@ -88,8 +88,8 @@ public void testSelectSettingsColumn() throws Exception {
 
     @Test
     public void testSetNonDynamicTableSetting() {
-        assertThrowsMatches(() -> execute("alter table settings_table set (\"soft_deletes.enabled\"='true')"),
-                     isSQLError(containsString("Can't update non dynamic settings [[index.soft_deletes.enabled]] for open indices"),
+        assertThrowsMatches(() -> execute("alter table settings_table set (codec = 'best_compression')"),
+                     isSQLError(containsString("Can't update non dynamic settings [[index.codec]] for open indices"),
```

Comment:
It's a bit unrelated to this change here, but the `index.` prefix of settings should be removed from the exception as it confuses when the used setting name does not contain it.

Reply:
done in https://github.com/crate/crate/pull/12640/commits/58cb6724f1cc2ede717672e60f0d442661eea3bb